src/main/java/org/sunbird/portal/department/model/AuditEvents.java

```diff
@@ -0,0 +1,148 @@
+package org.sunbird.portal.department.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+"eid",
+"ets",
+"ver",
+"mid",
+"actor",
+"context",
+"object",
+"edata"
+})
+
+public class AuditEvents {
+
+@JsonProperty("eid")
+private String eid;
+@JsonProperty("ets")
+private long ets;
+@JsonProperty("ver")
+private String ver;
+@JsonProperty("mid")
+private String mid;
+@JsonProperty("actor")
+private Actor actor;
+@JsonProperty("context")
+private Context context;
+@JsonProperty("object")
+private Object object;
+@JsonProperty("edata")
+private Edata edata;
+
+/**
+* No args constructor for use in serialization
+*
+*/
+public AuditEvents() {
+}
+
+/**
+*
+* @param actor
+* @param eid
+* @param edata
+* @param ver
+* @param ets
+* @param context
+* @param mid
+* @param object
+*/
+public AuditEvents(String eid, long ets, String ver, String mid, Actor actor, Context context, Object object, Edata edata) {
```

Comment:
Need to move this to another new package which should be separate for telemetry related classes.